src/toc.tsx

```diff
@@ -1,47 +1,82 @@
 // Copyright (c) Jupyter Development Team.
 // Distributed under the terms of the Modified BSD License.
 
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
 import { ActivityMonitor, PathExt } from '@jupyterlab/coreutils';
-
 import { IDocumentManager } from '@jupyterlab/docmanager';
-
 import { IRenderMimeRegistry } from '@jupyterlab/rendermime';
-
 import { Message } from '@phosphor/messaging';
-
 import { Widget } from '@phosphor/widgets';
+import { IHeading } from './utils/headings';
+import { Registry } from './registry';
+import { TOCTree } from './toc_tree';
 
-import { TableOfContentsRegistry } from './registry';
+/**
+ * Interface describing table of contents widget options.
+ *
+ * @private
```

Comment:
Why have you marked this as private? It is part of the public interface of the `TableOfContents` widget.

Reply:
You're right. Removed.